### ISSUE TITLE
[Designer- Card Element]: 'Drag' and 'Remove' control under 'Publish Adaptive Card Schema' is not accessible by keyboard.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7067]
https://microsoft.visualstudio.com/OS/_workitems/edit/37776442

**Comment by carlos-zamora** [comment]
We didn't hear back from the accessibility team regarding our concerns.

**Comment by carlos-zamora** [comment]
Of course! After we closed the issue, they reactivated it and responded haha. Here's their response:
> Here the issue is that if 'Drag' control is not keyboard accessible to the user then even any mechanism or shortcut key should be provided to the user. So, user can easily be able to drag the control. Hence Reactivating the bug.

**Comment by carlos-zamora** [comment]
This work item requires a major designer overhaul. For now, a proposal was written by Pankaj and is sitting in the Adaptive Cards OneNote.

Now that I'm thinking about it, might be worth posting that content as an issue, marking it as an epic, and linking any designer accessibility issues to it.

**Comment by carlos-zamora** [comment]
As suggested by Ajaz, closing this as a dupe of #7436.

**Comment by vagpt** [comment]
Since this issue is being tracked by https://github.com/microsoft/AdaptiveCards/issues/7436, hence adding A11yTTValidated tag.
